docs/basics.adoc

```diff
@@ -504,6 +504,39 @@ Connecting to Gremlin-Server (127.0.0.1:8182)... OK (connected to 127.0.0.1:8182
 Run gremlin.sh to connect.
 ----
 
+==== Running JanusGraph Server as a Daemon
+
+The JanusGraph https://github.com/JanusGraph/janusgraph/releases[release] also comes with an init script that can be found at `bin/gremlin-server.sh`.
+The script can also be run in the foreground by invoking `./bin/gremlin-server.sh` with an optional parameter of a Gremlin Server yaml file.
```

Comment:
Running JanusGraph as a daemon as described here only starts the Gremlin Server. The default pre-packaged distribution quickstart runs `janusgraph.sh` which starts Cassandra and Elasticsearch, so there's a bit of a disconnect from the previous section that should be documented here. When running JanusGraph as a daemon, it is expected that the operator will install/run the storage and indexing backends separately.

Reply:
I wonder whether it would make sense to convert `janusgraph.sh` into an init script that can be run as a service. I think that would give users the easiest way to get started with JanusGraph.
That could of course be done in a different PR. If we want to do that, then we could document here how `janusgraph.sh` can be used as a service and move the documentation about only running Gremlin Server into the JanusGraph Server chapter.

Reply:
The pre-packaged distribution discussion probably needs to move earlier anyway, like Chapter 3 Getting Started.

Reply:
Oh yes, I just assumed that this was part of the introduction because of the filename being `basics.adoc`, but then I agree that we should move in in a separate PR.

Reply:
I don't think it would take much for `janusgraph.sh` to work as an init script since it already supports `start/start/clean/status`.

Separately I had a different idea for easiest way to get started from a [medium article](https://medium.com/@patternrecognizer/how-to-add-a-try-in-play-with-docker-button-to-your-github-project-41cb65721e94) I read awhile back. I was just waiting for janusgraph-docker, and now the 0.2.3 release, to finish up.

I opened up issues for moving the `janusgraph.sh` docs, testing it as a init script, and creating a play with docker button. Still need to incorporate Jason's feedback about specifying that it's just running Gremlin server.

Reply:
Great idea with the play with docker button, I just want to node that I see `janusgraph.sh` more as something that can be used to actually set up a small JanusGraph database. We even used that in production in the beginning like that with a 3 node cluster.